lib/xcake/target.rb

```diff
@@ -165,7 +165,18 @@ def system_frameworks
     protected
 
     def default_system_frameworks_for(platform)
-      (platform == :ios) ? ['Foundation', 'UIKit'] : ['Cocoa']
+      case platform
+      when :ios
+        ['Foundation', 'UIKit']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%w` or `%W` for array of words.